### DISCUSSION TITLE
How to scrape from Fansly with new release

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [UltimaHoarder/UltimaScraper#1697] [General]
This question is for DC or preferably the community so I don't bother DC with dumb questions....... I know fansly was juat added....but I thought I setup the config and Auth correctly, it asked me to when I started it. But I do not see my fansly subs. Just onlyfans. Is it a separate scraper for fansly? Under Auth, I copy and pasted from the firefox cookie extension. Am I supposed to do anything more?

**Comment by DIGITALCRIMINAL** [comment]
Did you select fansly when choosing which site to scrape? You may have set it to automatically choose OnlyFans in the config

**Reply by J4k-The-Reaper** [reply]
So thats what I did wrong. Guess I'll have to open the config each time I want to scrape which ever site. Haha. You gonna maybe make it simpler? I know you said you only made it in 3 days and to expect bugs. But it is a feature that would be nice. I can manage changing it manually each time right now.

**Reply by DIGITALCRIMINAL** [reply]
You don't have to, you can leave the the option empty with quotes and the site options will show up. 
You can choose the sites just like how you choose which model to scrape.

**Reply by J4k-The-Reaper** [reply]
Oh....duh....I totally knew that. 🤦‍♂️🤥😳

Thanks

**Reply by J4k-The-Reaper** [reply]
Um, well, now when I chose fansly, it says Auth 404'd. I used the cookie extension to copy and paste into Auth. What did I do wrong? Do I need to input my email and password? This is what the Auth looks like:

![image_2021-09-18_062405](https://user-images.githubusercontent.com/35021417/133890427-3d174af4-4920-4455-86a9-4415975731be.png)

**Reply by J4k-The-Reaper** [reply]
now it just told me to do this....i must be doing something wrong....or i am just plain stupid!
![image_2021-09-18_063002](https://user-images.githubusercontent.com/35021417/133890596-e1d96932-2526-4df8-9283-5450f85a8cf3.png)

**Reply by DonaldTPP** [reply]
I think the problem you're encountering is because you're trying to use the OF Cookie helper on Fansly which doesn't work the same. You have to get the different AUTH key from Fansly a different way than when how you're getting your AUTH information from OnlyFans.

**Comment by Avnsx** [comment]
> This question is for DC or preferably the community so I don't bother DC with dumb questions....... I know fansly was juat added....but I thought I setup the config and Auth correctly, it asked me to when I started it. But I do not see my fansly subs. Just onlyfans. Is it a separate scraper for fansly? Under Auth, I copy and pasted from the firefox cookie extension. Am I supposed to do anything more?

Hey, I've recently stumbled over this question through google & I'm not sure how you would scrape fansly with DIGITALCRIMINAL's OnlyFans scraper. 
But I've basically written the equivalent of this OnlyFans scraper for fansly. If you still need to scrape fansly, here you go: https://github.com/Avnsx/fansly
